### DISCUSSION TITLE
string benchmark에 대한 실행 결과 질문

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#131] [Q&A] [answered]
컴파일 후 콘솔창과 실행후 나온 텍스트 파일 두개의 창을 캡처하는 형식으로 하면 되나요?

**Comment by Woon-2** [comment, marked as answer]
넴! 그런 과정으로 `string`의 실행 결과와 `vector`의 실행 결과를 따로 `commit`해주시기 바랍니다.
